src/modules/blocks/attendees/template.js

```diff
@@ -24,6 +24,7 @@ import './style.pcss';
  */
 
 const placeholder = __( 'Who\'s Attending?', 'events-gutenberg' );
+const subtitle    = __( 'How many people is attending this event', 'events-gutenberg' );
```

Comment:
"are", not "is"?

Reply:
Thanks! yeah, didn't pay much attention to the message. Was more obsessed with trying to get the post title and number of attendees. Fixed 👍